docs/introduction/how-tos/quickstart.md

```diff
@@ -1,6 +1,21 @@
 Quickstart Guide
 ================
 
+- [1. Compatibility](#compatibility)
+- [2. Deployment in the cloud](#deployment-in-the-cloud)
+- [3. Local Install](#local-install)
+
+Compatibility 
+----------------
+
+Local install is not feasible for MacBooks with M1/M2 Apple Sillicone chips. 
+In order to deploy the project on such devices, you have to procede with deployment in the cloud. 
+
+Deployment in the cloud
+------------------------
+
+Here, I will put all the steps how to deploy. (After, I successfully do so).
```

Comment:
Rather than leaving placeholder information, include changes that are able to merged and be relevant when deployed. The compatibility information is useful today and will save someone time by viewing it. The other items on the TOC and headings of empty or placeholder content can be safely removed for now.

Reply:
Just removed it!